### PR TITLE
additional info 8940 with tests

### PR DESCRIPTION
**Pull request by imeecuison**
## Description
Form to provide additional information for unemployability.
http://localhost:3001/disability-benefits/apply/form-526-all-claims/unemployability-additional-information

## Testing done
tests/pages/unemployabilityAdditionalInformation.unit.spec.jsx

## Screenshots

screen shot 2018-12-10 at 2 13 30 pm

## Acceptance criteria
- [ ] The user is able to populate the following:
- [ ] If there is any other information you would like to provide as part of your claim, please add it here. { text box }
NOTE: field is not required. 

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by desireeturner** [comment]
This PR is related to the following issue: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/14541

File: src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      If you‘ve left one or more jobs because of your service-connected
+      disabilities, please note that.
+    </p>
+  </div>
+);
+
+export const uiSchema = {
+  'ui:title': 'Individual Unemployability',
+  'ui:description': additionalInformationDescription,
+  remarks: {
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.js]: Hmm this seems _very_ similar to what we have for 781/781a, maybe we can use a different key or nest it underneath an `unemployability` object?

File: src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.js]: can we make this a `.jsx` file or move the content to the content folder?

File: src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+export const uiSchema = {
+  'ui:title': 'Individual Unemployability',
+  'ui:description': additionalInformationDescription,
+  remarks: {
+    'ui:title': ' ',
+    'ui:widget': 'textarea',
+    'ui:options': {
+      rows: 5,
+      maxLength: 3200,
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.js]: These are listed as requirements?

**Review by aaronjlech — CHANGES_REQUESTED** [review]
A few small things here!

File: src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.js (hunk 3 above)
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.js]: I just used examples of other textareas and these were the options used.

File: src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.js (hunk 1 above)
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.js]: I thought so too, but that's the key name in the schema.

Comment (aaronjlech) [reply src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.js]: gotcha, Can we nest it underneath the `unemployability` object like this:
```
export const schema = {
  type: 'object',
  properties: {
    unemployability: {
      type: 'object',
      properties: {
        hospitalizationHistory,
      },
    },
  },
};
```

Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.js]: changed!

**Review by aaronjlech — APPROVED** [review]
Looks good!

**Comment by imeecuison** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+const additionalInformationDescription = (
+  <div>
+    <p>
+      If there is any other information you would like to provide as part of
+      your claim, please add it here.
+    </p>
+    <p>
+      It‘s important to tell us why your service-connected disability \is the
```
Comment (rianfowler) [inline src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.jsx]: `\is` looks like a typo.

**Review by rianfowler — APPROVED** [review]
Looks good- one minor comment.

File: src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.jsx (hunk 4 above)
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.jsx]: Thank you!

**Pull request merged**